.mailmap

```diff
@@ -9,6 +9,7 @@
 Aaron Bassett <abassett@gmail.com>
 Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
```

Comment:
I think it would make sense to remove the
<pre>
Abhishek Lekshmanan <abhishek.lekshmanan@ril.com> <abhishek.lekshmanan@gmail.com>
</pre>
line.